dotCMS/src/main/java/com/dotmarketing/viewtools/StructuresWebAPI.java

```diff
@@ -276,4 +278,8 @@ public boolean haveViewPermision(Structure st, User user) throws DotDataExceptio
 	public boolean isFieldConstant(Field field){
 		return APILocator.getFieldAPI().isElementConstant(field);
 	}
+	
+	public boolean isFieldHidden(final Field field) throws DotDataException{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.exception.DotDataException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)